added Soplang v2.0.0

sharafdin opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/319686)

sharafdin committed:
added Soplang v2.0.0

stephengillie commented:
Automatic Validation ended with: 
> > No errors to post.

(Automated response - build 1176.)

sharafdin committed:
Change installer type to 'inno' and add switches

Updated installer type to 'inno' and added installer switches for silent installation.

sharafdin committed:
Merge branch 'microsoft:master' into master

sharafdin committed:
Fix version

sharafdin committed:
removed v2

stephengillie commented:
> Fix the version mismatch

Edit: Apolgies - I misunderstood what you meant.
